spec_finder.py

```diff
@@ -88,18 +88,18 @@ def main(refresh_spec: bool = False, diff_output: bool = False, limit_numbers: O
         if number in spec_map:
             txt = entry['text']
             if txt != spec_map[number]:
-                logging.info(f"{number} is bad")
+                logging.info("%s is bad", number)
                 bad_num += 1
                 if diff_output:
                     diff = difflib.ndiff([txt], [spec_map[number]])
                     logging.info('\n'.join([li for li in diff if not li.startswith(' ')]))
         else:
-            logging.info(f"{number} is defined in our tests, but couldn't find it in the spec")
+            logging.info("%s is defined in our tests, but couldn't find it in the spec", number)
 
     if missing:
         logging.info('In the spec, but not in our tests:')
         for m in sorted(missing):
-            logging.info(f"  {m}: {spec_map[m]}")
+            logging.info("%s: %s", m, spec_map[m])
```

Comment:
Just a note that this lost it's preceeding whitespace, which my fuzzy memory tells me wsa to help with visual parsing of the output.